new/dependency-update-cadence.md

```diff
@@ -0,0 +1,73 @@
+---
+RFC: unassigned
+Title: Dependency Update Cadence
+Author: Jennifer Davis <sigje@chef.io>
+Status: Draft
+Type: Process
+---
+
+# Dependency Update Cadence
+
+This RFC describes the Ruby, Cookbook and Ecosystem support cadence as an addendum to the [Chef OSS project versioning and release policies.](https://chef.github.io/chef-rfc/rfc086-chef-oss-project-policies.html) and the [cookbook release process](https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/RELEASE_PROCESS.MD).
+
+## Motivation
+
+    As a person who writes and shares community cookbooks, 
+    I want to drop old support for old client versions and use new client versions,
+    so that cookbook complexity can be reduced.
+
+    As a person who writes and shares community cookbooks,
+    I want to drop support for old versions of the Ruby language,
+    so that I don't have to support ruby versions that are no longer supported by the ruby community.
+
+    As a person who uses community cookbooks in my own recipes, 
+    I want to know the policy for cookbook support,
+    so I can understand the impact of delaying updating underlying dependencies.
+
+## Specification
+
+### Cookbook and Ecosystem Tooling Support
+
+Chef-managed community cookbooks should support at least the latest 6 months of [Chef Client](https://github.com/chef/chef/blob/master/CHANGELOG.md) versions. After 6 months, Chef-managed community cookbooks may
+elect to drop support for older Chef Client versions. This window does not reset on
+a major version release so that the prior major version track is supported for a 6 month window. Non-Chef-managed community cookbooks are encouraged to follow this policy.
+
+As an example, based on the [Chef OSS project versioning and release policies](https://chef.github.io/chef-rfc/rfc086-chef-oss-project-policies.html), in May if we release Chef Client 14.1.0, both 14.0 and 14.1 will be
+supported.  The version of 13 release 6 months prior should be 13.8 and will still
+be supported so that 6 versions will be considered current (13.8 through 13.11 plus
+14.0 and 14.1).  At that point community cookbooks may choose to start using 13.8
+features and drop support for versions prior to 13.7.
+
+Tooling external to cookbooks (cookstyle, chefspec, stove, foodcritic, halite,
+poise-hoist, etc) is similarly encouraged to follow this policy.
+
+### Ruby Cadence
+
+Since the Ruby language itself releases new minor versions over the Christmas holidays, the April major release of Chef Client should include the minor revision of Ruby which landed the prior Christmas.  Combined with the 6 month sliding window for cookbook support that also implies that when the prior major release of the client falls off of community cookbook support that the prior minor release of Ruby will also fall off of community cookbook support (including the cookstyle gem and related tooling).
```

Comment:
as per https://www.ruby-lang.org/en/news/2013/12/21/ruby-version-policy-changes-with-2-1-0/ this should be minor